src/components/Text/Text.scss

```diff
@@ -0,0 +1,73 @@
+@import '../variables';
+@import '../../../styles/mixins';
```

Comment:
Can we replace `@import` with `@use`?

Reply:
thank you for paying attention, I didn't know that this functionality would be dropped in future releases
https://github.com/sass/sass/blob/main/accepted/module-system.md#timeline